src/Domain/ReadySession/Model/ReadySessionBase64Image.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace StepupReadId\Domain\ReadySession\Model;
+
+final class ReadySessionBase64Image
+{
+    /** @var string  */
+    private $value;
+
+    public function __construct(string $base64Image)
+    {
+        $this->value = $base64Image;
```

Comment:
Consider adding input guards, for example, verify that the base64 image is a non-empty string. Not super important here. But a value object's state should be trustworthy.